src/content/platform-integration/ios/setup-ios.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Set up iOS development
+description: >-
+  Configure your development environment to
+  run, build, and deploy Flutter apps for iOS devices.
+short-title: Set up iOS
+---
+
+Learn how to set up your development environment
+to run, build, and deploy Flutter apps for iOS devices.
+
```

Comment:
This is missing the Rosetta instructions for Apple Silicon (ARM) Macs:
```
sudo softwareupdate --install-rosetta --agree-to-license
```
https://github.com/flutter/website/pull/7119#issuecomment-1124537969
https://github.com/flutter/flutter/issues/69157 (getting closer to done)